contrib/flatpak/template.py

```diff
@@ -0,0 +1,25 @@
+pkgname = "flatpak"
+pkgver = "1.14.3"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = ["--with-system-bubblewrap", "--with-system-dbus-proxy", "--with-system-helper-user=_flatpak",
```

Comment:
So there's `--with-system-helper-user=_flatpak` and `/usr/lib/sysusers.d/flatpak.conf` installed (containing `u _flatpak - "FLatpak system helper" -`), however afaik that file is a systemd specific thing and isn't used which means you'd instead need to set `system_users = ["_flatpak"]`